cmd/secrets-store-csi-driver/main.go

```diff
@@ -55,7 +55,7 @@ var (
 	logFormatJSON      = flag.Bool("log-format-json", false, "set log formatter to json")
 	providerVolumePath = flag.String("provider-volume", "/etc/kubernetes/secrets-store-csi-providers", "Volume path for provider")
 	// this will be removed in a future release
-	metricsAddr          = flag.String("metrics-addr", ":8095", "The address the metric endpoint binds to")
+	metricsAddr          = flag.String("metrics-addr", "8095", "The address the metric endpoint binds to")
```

Comment:
We should not have a configuration named `addr` when it actually accepts a port number as this will cause confusion.